.changes/v2023.7.25.md

```diff
@@ -0,0 +1,7 @@
+## [July 25, 2023](https://github.com/OpsLevel/opslevel-go/compare/v2023.7.17...v2023.7.25)
```

Comment:
Ahh sorry this is my bad - you went too far with changie - when adding new features or making refactors - in the golang tools all you need to do is `changie new` the rest of the process is automated in our release.  I know it differs from self-hosted but thats because we couldn't/didn't fully automated that on purpose.

Reply:
Fixed this